File: demo_java/src/main/java/com/techcentrix/sdk/demo_java/DemoApplication.java
package com.techcentrix.sdk.demo_java;

import android.app.Application;

import com.techcentrix.sdk.demo_java.util.ThemeHelper;

public class DemoApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        // TechCentrixSDK.setLoggingEnabled(true);
        ThemeHelper.applyThemeFromSharedPreferences(this);
    }
}

File: demo_kotlin/src/main/java/com/techcentrix/sdk/demo_kotlin/DemoApplication.kt
package com.techcentrix.sdk.demo_kotlin

import android.app.Application
import com.techcentrix.sdk.demo_kotlin.util.ThemeHelper

class DemoApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        // TechCentrixSDK.setLoggingEnabled(true)
        ThemeHelper.applyThemeFromSharedPreferences(this)
    }

}